qa/workunits/rbd/rbd-ggate.sh

```diff
@@ -133,18 +153,20 @@ used=`rbd -p ${POOL} --format xml du ${IMAGE} |
   $XMLSTARLET sel -t -m "//stats/images/image/used_size" -v .`
 [ "${used}" -lt "${provisioned}" ]
 
-# resize test
+echo  resize test
+# resizing the underlying image for a GEOM ggate will stop the
+# ggate process servicing the device. So we can resize and test 
+# the disapearence of the device
```

Comment:
I am still not happy with this comment. I am ok if you totally remove "XXX" comment below, because it is not true now (right now you just partially removed it which made things more confusing in my opinion). But this comment is confusing too: what it describes is true for the case when the kernel does not support resize, but this is not mentioned in the comment. Could you please update it or just remove.